core/new-gui/src/app/workspace/component/workflow-editor/workflow-editor.component.ts

```diff
@@ -976,8 +975,8 @@ export class WorkflowEditorComponent implements AfterViewInit {
    */
   private handleElementCopy(): void {
     Observable.fromEvent<ClipboardEvent>(document, 'copy')
-      .filter(event => (<HTMLElement>event.target).nodeName !== 'INPUT')
-      .subscribe(() => {
+    .filter(event => document.activeElement === document.body)
```

Comment:
why is this filtering `document.body`?

Reply:
document.body is the root document, which means there is no focused element on the entire page